themes/default/content/registry/packages/aws/how-to-guides/bucketv2-migration.md

```diff
@@ -0,0 +1,3708 @@
+---
+title: "BucketV2 Migration Guide"
+h1: "Migrating from aws.s3.Bucket to aws.s3.BucketV2"
+meta_desc: Practitioner level instructions for migrating from aws.s3.Bucket to aws.s3.BucketV2 resources
+layout: package
+---
+
+In the upcoming AWS Classic major release (v7), `aws.s3.Bucket` will be discontinued in favor of `BucketV2`. Users of
+`aws.s3.Bucket` resource are encouraged to migrate early. The migration requires a significant refactor to the source
```

Comment:
Maybe saying "Users of `aws.s3.Bucket` resource are encouraged to migrate early." is too aggressive. We don't actually want to push existing users to move yet I think.